blockstore/apps/bundles/admin.py

```diff
@@ -51,21 +54,32 @@ def snapshot_data(self, obj):
 
     def raw_summary(self, obj):
         """
-        Raw dump of BundleSnapshot data, though formatted more nicely.
+        Raw dump of Snapshot data, though formatted more nicely.
         """
         json_str = json.dumps(
-            obj.snapshot(), cls=BundleJSONEncoder, indent=4, sort_keys=True
+            obj.snapshot(), cls=BundleDataJSONEncoder, indent=4, sort_keys=True
         )
         return format_html("<pre>{}</pre>", json_str)
 
     def has_add_permission(self, request):
         return False
 
 
+class DraftInline(admin.StackedInline):
+    """Draft inline view. """
+    model = Draft
+    classes = ['collapse']
+
+    def get_max_num(self, request, obj=None, **kwargs):
+        if obj is None:
+            return 3
+        return max(obj.drafts.count(), 3)
```

Comment:
This seems odd to me. Are there scenarios where one might want to create multiple drafts at the same time?

Reply:
I think that imports will want their own private draft space.

Reply:
Right, I just meant that I don't foresee someone manually creating Studio + Import draft spaces from the django admin UI at the same time.